xbmc/utils/Color.h

```diff
@@ -15,6 +15,15 @@ namespace UTILS
 
   typedef uint32_t Color;
 
+  struct ColorEx
+  {
+    std::string hex;
```

Comment:
Does this need to be a std::string since it only refers to an hex value. Can't the color typedef be used instead?

Reply:
removed